CoralCommon/src/MonitoringEvent.cpp

```diff
@@ -22,31 +22,6 @@ namespace coral
       {
```

Comment:
@guitargeek , shouldn;t we have `m_data( std::move(data))` here?

Reply:
The consructor takes a raw pointer `Payload* data` that is passed to the constructor of `std::unique_ptr<Payload> m_data`. Raw pointers don't need to me moved. If the `data` passed the the `Record` constructor would be a smart pointer already, you would be right though.